src/features/products/components/Navigation/Navigation.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { FC } from "react";
 import { Link } from "react-router-dom";
-import "./ListNavigation.scss";
 
-const ListNavigation: FC = () => {
+import "./Navigation.scss";
+
+const Navigation: FC = () => {
   return (
     <ul className="list-nav">
```

Comment:
Simplify

Reply:
Remove FC from all components without props

Reply:
I meant to remove the unnecessary return

Reply:
Removed